domains/cursedscrubdaddygang.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "repo": "https://github.com/QuantumMapleQC/CursedScrubDaddyGang"
+        "username": "quantummapleqc"
+        "Discord": "herricksom"
+    },
+    "record": {
+        "CNAME": "quantummapleqc.github.io/CursedScrubDaddyGang/"
```

Comment:
CNAME is invalid. CNAME should be a hostname without any additional components like protocols or file paths.